src/syft/lib/tenseal/__init__.py

```diff
@@ -13,9 +13,13 @@
 from ...ast import add_modules
 from ...ast.globals import Globals
 from ..misc.union import UnionGenerator
-from .ckks_vector import CKKSVector  # noqa: 401
 from .context import ContextWrapper  # noqa: 401
 
+from .ckks_vector import CKKSVector  # noqa: 401
```

Comment:
Q: The ```noqa: 401``` isn't it used for modules that are unused?
Can't we delete it since we are using the modules?

Reply:
To do that I think the code should change from:
```
        ("tenseal.CKKSTensor", "tenseal.CKKSTensor", ts.CKKSTensor),
```

```
        ("tenseal.CKKSTensor", "tenseal.CKKSTensor", CKKSTensor),
```

Then the `# noqa: 401` can be removed.

Reply:
Oh, I see they're actually different. I wonder if this should be named differently to prevent confusion, not a big deal either way. I guess we normally call these wrappers:
```
class CKKSVector(StorableObject):
```

```
class CKKSVectorWrapper(StorableObject):
```

Since the serde is handled transparently using the serializable_wrapper_type property on the real class the user should never interact with these Wrappers anyway.